datamade/just-spaces
# Update data import pipeline for new geographies

## Overview

This PR updates the data import pipeline to add new CensusRegions and CensusBlockgroups to the app. It also makes some small tweaks to the display of the CensusArea widget map to allow display of different regions and fix a few bugs.

Once this PR is landed and Tyler has approved the general approach, we can go ahead and add the remaining ~15 cities.

Connects #220. 

## Testing Instructions

* Clean out your local data files with `docker-compose run --rm make clean`
* Grab new data with `docker-compose run --rm make all` (this may take 5-10 minutes)
* Run a fresh import with `python manage.py import_data`
* Run the app and navigate to http://localhost:8000/census-areas/
* Try creating new CensusAreas in the new regions
* Log in as a staff user (non-superuser) and confirm that the Create view for CensusAreas hides the Agency checkbox from you
* Test that the new CensusAreas can be used to filter census data in the DAD
